code_fragments/SCALLOP_CAMS_GC_LA.do

```diff
@@ -0,0 +1,68 @@
+***This file lables all CAMS trips (including orphans) as limited Access or GC.
+
+
+ clear all
+
+jdbc connect , jar("$jar")  driverclass("$classname")  url("$nefscdb1_url")  user("$myuid") password("$nefscdb1pwd")
+ 
+jdbc load, exec ("select a.CAMSID, d.ACTIVITY_CODE_1, d.ACCESSAREA from CAMS_GARFO.CAMS_LAND a full outer join CAMS_GARFO.CAMS_subtrip d on a.CAMSID=d.CAMSID where a.ITIS_GROUP1 like '%SCALLOP%' and a.YEAR between $startyear and $endyear ") 
+ 
+save "${my_datadir}/intermediate/CAMS_SCALLOP_TRIPS.dta", replace 
+ 
+ 
+ ****load the orphans 
+  clear all
+
+jdbc connect , jar("$jar")  driverclass("$classname")  url("$nefscdb1_url")  user("$myuid") password("$nefscdb1pwd")
+ 
+jdbc load, exec ("select CAMSID, ACTIVITY_CODE_1, ACCESSAREA from CAMS_GARFO.CAMS_VTR_ORPHANS_SUBTRIP where MAIN_SPP_GRP like '%SCA%' and YEAR between $startyear and $endyear ") 
+ 
+ append using "${my_datadir}/intermediate/CAMS_SCALLOP_TRIPS.dta", replace 
+ 
+
+ split ACTIVITY_CODE_1, p(-)
+ 
+ gen activity_code = ACTIVITY_CODE_11 +"-"+ ACTIVITY_CODE_12
+ 
+ **label all Limited access trips 
+ gen LA= 1 if activity_code =="SES-SAA" |activity_code =="SES-SCA" 
+ replace LA = 0 if missing(LA)
+ 
+**label all general category trips 
+ gen GC= 1 if activity_code =="SES-SCG" 
+  replace GC = 0 if missing(GC)
+  
+ *** rename access area
+ 
+ rename ACCESSAREA access_area
+ replace access_area = "1" if access_area=="AA"
+  replace access_area = "0" if access_area!="1"
```

Comment:
I don't know enough about how cams is put together, but did this give the same results as
```
gen access_area2=activity_code=="SES-SAA"
```
for the LA vessels? I'm not sure how to get the access area fishing for the GC vessels.

Reply:
There is an access area in CAMS so that is what I used to define access_area. Does that make sense?

Reply:
Sure.